contracts/ubeswap/contracts/uniswapv2/interfaces/IUbeswapRouter.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This is the interface Ubeswap uses on its UbeswapMoolaRouter; just a subset of the UniswapV2Router02 interface.